content/pt-br/docs/reference/glossary/app-container.md

```diff
@@ -0,0 +1,20 @@
+---
+título: Contêiner de aplicações
+id: app-container
+data: 2023-01-24
+full_link:
+short_description: >
+   Um contêiner usado para executar parte de uma carga de trabalho. Pode ser comparado com um contêiner init.
+
+akaa:
+tags:
+- workload
+---
+  Contêineres de aplicações (ou app containers) são os {{< glossary_tooltip text="containers" term_id="container" >}} de um {{< glossary_tooltip text="pod" term_id="pod" >}} que são iniciados após qualquer {{< glossary_tooltip text="init containers" term_id="init-container" >}} terem sido concluídos.
```

Comment:
Some bits here:
* `glossary_tooltip` `text` properties should be translated.
* We usually stylize the Kubernetes objects with the first letter capitalized. Thus, Pod should be rendered with a capital P.